lib/call.js

```diff
@@ -112,6 +112,31 @@ var Call = function (client) {
 		return this.update(callId, { state : "active" }).asCallback(callback);
 	};
 
+	/**
+	 * Transfer a call
+	 * @param {Object} params Parameters for transfering of the call
+	 * @param {String} params.transferTo Phone number or SIP address that the call
+	 * is going to be transferred to.
+	 * @param {String} params.transferCallerId The caller id that will be used when the call is transferred
+	 * @param {Object} params.whisperAudio Audio to be played to the caller that the call will be transferred to.
+	 * @param {Function} [callback] Callback with the transfered call
+	 * @return {CallResponse} A promise for the transfered call
+	 */
+	this.transfer = function (callId, params, callback) {
+		params.state = "transferring";
+		return client.makeRequest({
```

Comment:
Why make the request here instead of using the update function like [answer](https://github.com/bandwidthcom/node-bandwidth/pull/68/files#diff-5356829d8a075e5e6cb4bb402c48f16eL111) ?